cov.el

```diff
@@ -260,6 +260,26 @@ that filename. Otherwise search for the first matching pattern in
                                 cov-lcov-patterns)))
     (when lcov-file (cons lcov-file 'lcov))))
 
+(defun cov--locate-gcov-cmake (file-dir file-name)
+  "Locate a gcov coverage file from FILE-DIR for FILE-NAME.
+This works with CMake-based projects, by constructing the path to the `.gcov'
+file from object file's path extracted from the \"compile_commands.json\"."
+  (when-let* ((root (project-root (project-current)))
+              (compile-commands (expand-file-name "compile_commands.json" root)))
+    (when (file-exists-p compile-commands)
+      (when-let* ((file-cmd (cl-find-if
```

Comment:
`when-let*` was introduced in Emacs 26 (as far as I can tell).  cov.el supports Emacs 24.4 onward. Please use other means to do this.

Reply:
Thanks for the feedback, I fixed this!